lib/msf/core/opt_proxies.rb

```diff
@@ -0,0 +1,33 @@
+# -*- coding: binary -*-
+
+module Msf
+
+###
+#
+# Proxies option
+#
+###
+class OptProxies < OptBase
+
+  def type
+    'proxies'
+  end
+
+  def validate_on_assignment?
+    true
+  end
+
+  def normalize(value)
+    value
+  end
+
+  def valid?(value=self.value, check_empty: true, datastore: nil)
```

Comment:
Will extract out to a separate PR; I noticed it's possible to assign invalid values to the `proxies` datastore option. This should tighten things up, but the UX still isn't great IMO